layers/+lang/extempore/packages.el

```diff
@@ -0,0 +1,97 @@
+;;; packages.el --- extempore layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2020 Ben Swift
+;;
+;; Author: Ben Swift <ben@benswift.me>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;; See the Spacemacs documentation and FAQs for instructions on how to implement
+;; a new layer:
+;;
+;;   SPC h SPC layers RET
+;;
+;;
+;; Briefly, each package to be installed or configured by this layer should be
+;; added to `extempore-packages'. Then, for each package PACKAGE:
+;;
+;; - If PACKAGE is not referenced by any other Spacemacs layer, define a
+;;   function `extempore/init-PACKAGE' to load and initialize the package.
+
+;; - Otherwise, PACKAGE is already referenced by another Spacemacs layer, so
+;;   define the functions `extempore/pre-init-PACKAGE' and/or
+;;   `extempore/post-init-PACKAGE' to customize the package as it is loaded.
+
+;;; Code:
+
+(defconst extempore-packages
+  '(extempore-mode)
+  "The list of Lisp packages required by the extempore layer.
+
+Each entry is either:
```

Comment:
Please remove these comments these are only for guidance during the creation and should not stay in finished files.